packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -3,10 +3,28 @@ import 'jest-styled-components';
 import deepClone from 'ramda/src/clone';
 import renderWithHelmet from './renderWithHelmet';
 
-const createSnapshot = (component, done) => {
-  // select the first child to remove the pointless wrapping div from snapshots
-  const removeWrappingDiv = container => container.firstChild;
-  renderWithHelmet(component).then(({ container }) => {
+// select the first child to remove the pointless wrapping div from snapshots
+const removeWrappingDiv = container => container.firstChild;
+
+export const shouldMatchSnapshot = (title, component) => {
+  it(title, done => {
+    renderWithHelmet(component).then(({ container }) => {
+      const hasOneChild = container.children.length === 1;
```

Comment:
Struggling to find a nice way to reuse
```
    const hasOneChild = container.children.length === 1;
    /*
     * if the container has more than one child then it's a component that uses a
     * fragment at the top level so we should not select the first child because it
     * wouldn't snapshot the whole component
     */
    expect(
      hasOneChild ? removeWrappingDiv(container) : container,
    ).toMatchSnapshot();
```

Because `matchSnapshotAsync` needs to return `renderWithHelmet`, whereas `shouldMatchSnapshot` doesn't need to return it and also needs done(); at the end of the `renderWithHelmet` function.

Reply:
Actually I'm just being slow ... commit incoming